source/extensions/clusters/redis/redis_cluster.cc

```diff
@@ -187,12 +187,16 @@ ProcessCluster(const NetworkFilters::Common::Redis::RespValue& value) {
 
   std::string address = array[0].asString();
   bool ipv6 = (address.find(':') != std::string::npos);
-  if (ipv6) {
-    return std::make_shared<Network::Address::Ipv6Instance>(address, array[1].asInteger());
+  try {
+    if (ipv6) {
+      return std::make_shared<Network::Address::Ipv6Instance>(address, array[1].asInteger());
+    }
+    return std::make_shared<Network::Address::Ipv4Instance>(address, array[1].asInteger());
+  } catch (const EnvoyException& ex) {
+    ENVOY_LOG(warn, "Invalid ip address in CLUSTER SLOTS response: {}", ex.what());
```

Comment:
I would put this at debug as this comes from external input and might spam at high rate. Also do you want  stat of some type?

Reply:
Ok, I'll change this to debug.  This would be tracked by cluster's update_failure stat.  I think that's probably sufficient for now.